File: SofiForce.Web/ClientApp/src/app/core/Models/EntityModels/PromtionCriteriaSalesManAttrModel.ts
export interface PromtionCriteriaSalesManAttrModel {
    salesManAttributeId: number;
    salesManAttributeCode: string;
    salesManAttributeNameAr: string;
    salesManAttributeNameEn: string;
    salesManAttributeDesc: string;
    isCustom: boolean | null;
    isActive: boolean | null;
    canEdit: boolean | null;
    canDelete: boolean | null;
    displayOrder: number | null;
    cBy: number | null;
    cDate: Date | null;
    eBy: number | null;
    eDate: Date | null;
}

File: SofiForce.Models/Models/EntityModels/PromtionCriteriaSalesManAttrModel.cs
using System;

namespace Models
{
    public class PromtionCriteriaSalesManAttrModel
    {
        public int SalesManAttributeId { get; set; }
        public string SalesManAttributeCode { get; set; }
        public string SalesManAttributeNameAr { get; set; }
        public string SalesManAttributeNameEn { get; set; }
        public string SalesManAttributeDesc { get; set; }
        public bool? IsCustom { get; set; }
        public bool? IsActive { get; set; }
        public bool? CanEdit { get; set; }
        public bool? CanDelete { get; set; }
        public int? DisplayOrder { get; set; }
        public int? CBy { get; set; }
        public DateTime? CDate { get; set; }
        public int? EBy { get; set; }
        public DateTime? EDate { get; set; }
    }
}
